spring-web/spring-web.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
 	optional("io.netty:netty-all")
 	optional("io.projectreactor.ipc:reactor-netty")
 	optional("org.apache.tomcat.embed:tomcat-embed-core:${tomcatVersion}")
+	optional("org.eclipse.jetty:jetty-reactive-httpclient:0.9.2")
```

Comment:
Version 1.0.0 with RS TCK implemented is available in Maven Central.

Reply:
Updated